backend/common/src/main/java/ai/verta/modeldb/common/artifactStore/storageservice/ArtifactStoreService.java

```diff
@@ -100,7 +100,7 @@ default String getUrl(
         UriComponentsBuilder.newInstance().scheme(scheme).host(finalHost).path(endpoint);
     for (Map.Entry<String, Object> queryParam : parameters.entrySet()) {
       // Adding the filename to the path to enable file saved with that name
-      if (queryParam.getKey().equals(ModelDBConstants.FILENAME)) {
+      if (queryParam.getKey().equals(CommonConstants.FILENAME)) {
```

Comment:
please rename constant to `FILENAME_QUERY_PARAM`

Reply:
Fixed